lib/api/src/modules/notifications.ts

```diff
@@ -3,7 +3,15 @@ import { ModuleFn } from '../index';
 export interface Notification {
   id: string;
   link: string;
-  content: string;
+  content: {
+    headline: string;
+    subHeadline?: string;
+  };
+
+  icon?: {
+    name: string;
```

Comment:
Should this be a enum?

Reply:
We had that before but it caused a weird circular dependency, so we loosened it to get around that.

Reply:
ok